### PR TITLE
[SPARK-33397][YARN][DOC] Fix generating md to html for available-patterns-for-shs-custom-executor-log-url

### PR DESCRIPTION
**Pull request by yaooqinn**
### What changes were proposed in this pull request?

1. replace `{{}}`  with `&#123;&#123;&#125;&#125;`
2. using `<code></code>` in td-tag

### Why are the changes needed?

to fix this.
image

### Does this PR introduce _any_ user-facing change?

yes, you will see the correct online doc with this change 

image

### How was this patch tested?

shown as the above pic via jekyll serve.

### CODE REVIEW

**Comment by yaooqinn** [comment]
cc @HyukjinKwon @cloud-fan @maropu @HeartSaVioR thanks

File: docs/running-on-yarn.md [hunk 1]
```diff
@@ -584,48 +584,48 @@ To use a custom metrics.properties for the application master and executors, upd
...
-      <td>`stdout`, `stderr`.</td>
+      <td>&#123;&#123;FILE_NAME&#125;&#125;</td>
+      <td><code>stdout</code>, <code>stderr</code>.</td>
     </tr>
 </table>
 
 For example, suppose you would like to point log url link to Job History Server directly instead of let NodeManager http server redirects it, you can configure `spark.history.custom.executor.log.url` as below:
 
- `{{HTTP_SCHEME}}<JHS_HOST>:<JHS_PORT>/jobhistory/logs/{{NM_HOST}}:{{NM_PORT}}/{{CONTAINER_ID}}/{{CONTAINER_ID}}/{{USER}}/{{FILE_NAME}}?start=-4096`
+<code>&#123;&#123;HTTP_SCHEME&#125;&#125;&lt;JHS_HOST&gt;:&lt;JHS_PORT&gt;/jobhistory/logs/&#123;&#123;NM_HOST&#125;&#125;:&#123;&#123;NM_PORT&#125;&#125;/&#123;&#123;CONTAINER_ID&#125;&#125;/&#123;&#123;CONTAINER_ID&#125;&#125;/&#123;&#123;USER&#125;&#125;/&#123;&#123;FILE_NAME&#125;&#125;?start=-4096</code>
```
Comment (maropu) [inline docs/running-on-yarn.md:626]: we need to change `<` ->`&lt;` and `>` -> `&gt;`?

Comment (yaooqinn) [reply docs/running-on-yarn.md:626]: yes. otherwise，they become tags inside codeblock

**Review by cloud-fan — APPROVED** [review]
LGTM if you have generated the doc and verified locally.

**Comment by maropu** [comment]
LGTM (I've checked the generated page manually)

**Comment by maropu** [comment]
Merged to master/branch-3.0. Thanks!

**Comment by HeartSaVioR** [comment]
Late +1. Thanks for fixing!

**Pull request closed without merging**